src/gp_engine.hpp

```diff
@@ -35,28 +39,50 @@ class GPEngine {
   void SetTransitionAndDerivativeMatricesToHaveBranchLength(double branch_length);
   void SetTransitionMatrixToHaveBranchLengthAndTranspose(double branch_length);
   const Eigen::Matrix4d& GetTransitionMatrix() { return transition_matrix_; };
+  NucleotidePLV GetPLV(size_t idx) { return plvs_[idx]; }
```

Comment:
Is this used? Could it be const?

Reply:
Not used. I put it there to test code but it's no longer needed. Removed it.